client/task-list/tasks/payments/methods.js

```diff
@@ -95,6 +96,32 @@ export function getPaymentMethods( {
 				options.woocommerce_stripe_settings.enabled === 'yes',
 			optionName: 'woocommerce_stripe_settings',
 		},
+		{
+			key: 'mercadopago',
+			title: __(
+				'Mercado Pago Checkout Pro & Custom',
+				'woocommerce-admin'
+			),
+			content: (
+				<>
+					{ __(
+						'Accept credit and debit cards, offline (cash or bank transfer) and logged-in payments with money in Mercado Pago. Safe and secure payments with the leading payment processor in LATAM.',
+						'woocommerce-admin'
+					) }
+				</>
+			),
+			before: (
+				<img src={ wcAssetUrl + 'images/mercadopago.png' } alt="" />
```

Comment:
This image didn't exist for me. Does it need to be committed in the PR?

Reply:
Hi @adrianduffell,

> This image didn't exist for me.

Do you have the latest `trunk` version in the `WooCommerce` plugin you're using to test?
The image was merged a few days ago in [this PR](https://github.com/woocommerce/woocommerce/pull/29365).

> Does it need to be committed in the PR?

I added the image there to follow the pattern we are using for the `.png` logos in the payments methods (all of them are in the WooCommerce folder `assets/images/`).

Do you think we should start to store these images in a local folder?

Reply:
@octaedro Ah, I had core 5.1 installed instead of the latest `trunk`. I guess this exposes an issue with the pattern where the image might not be available to WC Admin due to the version of core in use.

I think it would work well to have these images in a local WC Admin folder. Maybe there is a good reason to have them in core though?

Reply:
> I think it would work well to have these images in a local WC Admin folder. Maybe there is a good reason to have them in core though?

Good idea! I couldn't think of any good reason to keep these images in `core`, so I put this PNG under `onboarding` in the commit 4044c5d.